src/website/app/demos/Form/examples/FormFieldset/basic.js

```diff
@@ -8,7 +8,7 @@ export default {
   title: 'Basic Usage',
   description: `Wrap any number of [FormFields](/components/form-field) in a FormFieldset
 to semantically group them. A brief, descriptive legend is especially useful for
-users of Assistive Technology (AT), such as a screen reader.`,
+users of screen readers and other assistive technologies.`,
```

Comment:
Should we leave in both or just say "assistive technologies" here?

Reply:
or maybe "user of assistive technologies like screen readers."